tests/cancun/eip4844_blobs/test_blob_txs.py

```diff
@@ -1032,3 +1033,25 @@ def test_blob_type_tx_pre_fork(
         blocks=blocks,
         genesis_environment=Environment(),  # `env` fixture has blob fields
     )
+
+
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize("destination_account", [None], ids=[""])
+@pytest.mark.parametrize("tx_calldata", [Initcode(deploy_code=Op.SSTORE(0, 1))], ids=[""])
+@pytest.mark.parametrize("tx_gas", [100_000], ids=[""])
+@pytest.mark.parametrize("tx_error", ["contract creation not possible in blob tx"], ids=[""])
```

Comment:
I just made up this `tx_error` message (didn't find one). But, it could be that it's not checked/used due to the `if` in L279:

https://github.com/danceratopz/execution-spec-tests/blob/a9f31ffd803502b13043227589d517aaa128b0a3/tests/cancun/eip4844_blobs/test_blob_txs.py#L251-L279